stable/jaeger-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+jaeger-operator is installed.
+
+
+Check the jaeger-operator logs
+  export POD=$(kubectl get pods -l app={{ template "jaeger-operator.fullname" . }} --namespace {{ .Release.Namespace }} --output name)
```

Comment:
Should be `export POD=$(kubectl get pods-l app.kubernetes.io/instance={{ .Release.Name }} -lapp.kubernetes.io/name=jaeger-operator --namespace {{ .Release.Namespace }} --output name)`

Reply:
done